core/templates/pages/exploration-editor-page/editor-tab/services/interaction-details-cache.service.ts

```diff
@@ -58,7 +58,7 @@ export class InteractionDetailsCacheService {
     };
   }
 
-  get(interactionId: string): InteractionCustomizationArgs {
+  get(interactionId: string): InteractionCustomizationArgs | null {
```

Comment:
Hm, interesting. I thought with stricter checks, we would actually be removing "| null" rather than adding it. I might be missing something.  @vojtechjelinek / @nishantwrp is this expected?

Reply:
1. We wont be removing nulls. It's that if null is needed it needs to be defined explicitly.

2. Here in the function we are returning null in a if condition. So, I think it's fine.

Reply:
Strict checks mean that `null` values need to be explicitely mentioned when they are needed.

Reply:
Ah got it. Thanks @nishantwrp and @vojtechjelinek for explaining!